[2025-03-31T21:30:53.616+0000] {processor.py:186} INFO - Started process (PID=971) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:30:53.617+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:30:53.620+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.619+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:30:53.642+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:30:53.796+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.795+0000] {override.py:1912} INFO - Created Permission View: can edit on DAG:premier_league_data_pipeline
[2025-03-31T21:30:53.813+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.813+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG:premier_league_data_pipeline
[2025-03-31T21:30:53.824+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.824+0000] {override.py:1912} INFO - Created Permission View: can read on DAG:premier_league_data_pipeline
[2025-03-31T21:30:53.836+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.836+0000] {override.py:1912} INFO - Created Permission View: menu access on DAG Run:premier_league_data_pipeline
[2025-03-31T21:30:53.847+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.846+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG Run:premier_league_data_pipeline
[2025-03-31T21:30:53.859+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.858+0000] {override.py:1912} INFO - Created Permission View: can read on DAG Run:premier_league_data_pipeline
[2025-03-31T21:30:53.870+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.869+0000] {override.py:1912} INFO - Created Permission View: can create on DAG Run:premier_league_data_pipeline
[2025-03-31T21:30:53.871+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.870+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:30:53.888+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.888+0000] {dag.py:3262} INFO - Creating ORM DAG for premier_league_data_pipeline
[2025-03-31T21:30:53.908+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:53.908+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:30:53.933+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.329 seconds
[2025-03-31T21:30:57.660+0000] {processor.py:186} INFO - Started process (PID=972) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:30:57.662+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:30:57.665+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:57.664+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:30:57.686+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:30:57.705+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:57.704+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:30:57.744+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:30:57.744+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:30:57.776+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.125 seconds
[2025-03-31T21:31:28.131+0000] {processor.py:186} INFO - Started process (PID=1036) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:31:28.133+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:31:28.135+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:31:28.135+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:31:28.182+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:31:28.268+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:31:28.267+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:31:28.367+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:31:28.367+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:31:28.497+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.378 seconds
[2025-03-31T21:32:00.812+0000] {processor.py:186} INFO - Started process (PID=1092) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:32:00.816+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:32:00.821+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:32:00.821+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:32:00.874+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:32:01.000+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:32:00.989+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:32:01.107+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:32:01.107+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:32:01.160+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.362 seconds
[2025-03-31T21:32:31.980+0000] {processor.py:186} INFO - Started process (PID=1155) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:32:31.984+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:32:31.987+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:32:31.987+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:32:32.014+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:32:32.053+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:32:32.053+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:32:32.114+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:32:32.113+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:32:32.156+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.191 seconds
[2025-03-31T21:33:02.740+0000] {processor.py:186} INFO - Started process (PID=1215) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:33:02.741+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:33:02.744+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:33:02.743+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:33:02.775+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:33:02.813+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:33:02.812+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:33:02.856+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:33:02.856+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:33:02.884+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.155 seconds
[2025-03-31T21:33:33.463+0000] {processor.py:186} INFO - Started process (PID=1278) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:33:33.464+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:33:33.467+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:33:33.467+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:33:33.493+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:33:33.527+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:33:33.526+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:33:33.574+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:33:33.574+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:33:33.604+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.152 seconds
[2025-03-31T21:34:03.729+0000] {processor.py:186} INFO - Started process (PID=1342) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:34:03.732+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:34:03.737+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:34:03.736+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:34:03.832+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:34:03.929+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:34:03.928+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:34:04.005+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:34:04.005+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:34:04.053+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.341 seconds
[2025-03-31T21:34:34.281+0000] {processor.py:186} INFO - Started process (PID=1405) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:34:34.284+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:34:34.289+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:34:34.288+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:34:34.321+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:34:34.383+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:34:34.382+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:34:34.440+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:34:34.440+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:34:34.490+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.222 seconds
[2025-03-31T21:35:04.598+0000] {processor.py:186} INFO - Started process (PID=1468) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:35:04.600+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:35:04.604+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:35:04.603+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:35:04.642+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:35:04.683+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:35:04.683+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:35:04.741+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:35:04.741+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:35:04.789+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.205 seconds
[2025-03-31T21:35:35.767+0000] {processor.py:186} INFO - Started process (PID=1532) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:35:35.768+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:35:35.771+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:35:35.771+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:35:35.799+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:35:35.831+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:35:35.831+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:35:35.878+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:35:35.877+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:35:35.907+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.148 seconds
[2025-03-31T21:36:06.581+0000] {processor.py:186} INFO - Started process (PID=1596) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:36:06.583+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:36:06.587+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:36:06.587+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:36:06.625+0000] {processor.py:925} INFO - DAG(s) 'premier_league_data_pipeline' retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:36:06.675+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:36:06.675+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-03-31T21:36:06.728+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:36:06.728+0000] {dag.py:4180} INFO - Setting next_dagrun for premier_league_data_pipeline to 2026-01-01 00:00:00+00:00, run_after=2027-01-01 00:00:00+00:00
[2025-03-31T21:36:06.769+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.204 seconds
[2025-03-31T21:36:54.849+0000] {processor.py:186} INFO - Started process (PID=1648) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:36:54.859+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:36:54.865+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:36:54.861+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:36:54.915+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:36:54.887+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:36:54.917+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:36:54.998+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.185 seconds
[2025-03-31T21:37:25.173+0000] {processor.py:186} INFO - Started process (PID=1710) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:37:25.178+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:37:25.194+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:37:25.193+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:37:25.216+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:37:25.212+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:37:25.218+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:37:25.331+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.170 seconds
[2025-03-31T21:37:56.044+0000] {processor.py:186} INFO - Started process (PID=1772) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:37:56.046+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:37:56.056+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:37:56.055+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:37:56.076+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:37:56.074+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:37:56.079+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:37:56.130+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.096 seconds
[2025-03-31T21:38:27.181+0000] {processor.py:186} INFO - Started process (PID=1836) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:38:27.183+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:38:27.188+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:38:27.188+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:38:27.302+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:38:27.297+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:38:27.305+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:38:27.380+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.225 seconds
[2025-03-31T21:39:48.935+0000] {processor.py:186} INFO - Started process (PID=1883) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:39:48.949+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:39:48.998+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:39:48.998+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:39:49.116+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:39:49.107+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:39:49.132+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:39:49.362+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.498 seconds
[2025-03-31T21:40:24.054+0000] {processor.py:186} INFO - Started process (PID=1945) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:40:24.058+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:40:24.064+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:40:24.063+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:40:24.095+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:40:24.088+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:40:24.098+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:40:24.284+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.258 seconds
[2025-03-31T21:42:03.533+0000] {processor.py:186} INFO - Started process (PID=1976) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:42:03.542+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:42:03.556+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:42:03.549+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:42:03.616+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:42:03.600+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:42:03.621+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:42:05.637+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 2.126 seconds
[2025-03-31T21:43:00.372+0000] {processor.py:186} INFO - Started process (PID=2032) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:43:00.375+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:43:00.383+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:43:00.380+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:43:00.394+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:43:00.392+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:43:00.396+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:43:00.441+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.084 seconds
[2025-03-31T21:43:44.190+0000] {processor.py:186} INFO - Started process (PID=2056) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:43:44.193+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:43:44.196+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:43:44.195+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:43:44.222+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:43:44.216+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:43:44.225+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:43:44.282+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.116 seconds
[2025-03-31T21:44:14.543+0000] {processor.py:186} INFO - Started process (PID=2115) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:44:14.545+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:44:14.547+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:44:14.546+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:44:14.570+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:44:14.565+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:44:14.572+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:44:14.611+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.079 seconds
[2025-03-31T21:44:45.005+0000] {processor.py:186} INFO - Started process (PID=2179) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:44:45.006+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:44:45.013+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:44:45.010+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:44:45.025+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:44:45.021+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:44:45.029+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:44:45.075+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.086 seconds
[2025-03-31T21:45:15.903+0000] {processor.py:186} INFO - Started process (PID=2243) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:45:15.904+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:45:15.909+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:45:15.908+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:45:15.923+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:45:15.921+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:45:15.925+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:45:16.005+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.120 seconds
[2025-03-31T21:45:47.320+0000] {processor.py:186} INFO - Started process (PID=2307) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:45:47.372+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:45:47.488+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:45:47.472+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:45:47.783+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:45:47.677+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:45:48.025+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:45:49.139+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 1.966 seconds
[2025-03-31T21:49:03.151+0000] {processor.py:186} INFO - Started process (PID=2317) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:49:03.201+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:49:03.384+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:49:03.374+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:49:04.163+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:49:04.145+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:49:04.167+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:49:05.066+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 2.039 seconds
[2025-03-31T21:49:35.457+0000] {processor.py:186} INFO - Started process (PID=2380) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:49:35.460+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:49:35.466+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:49:35.464+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:49:35.479+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:49:35.476+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:49:35.482+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:49:35.530+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.089 seconds
[2025-03-31T21:50:05.641+0000] {processor.py:186} INFO - Started process (PID=2444) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:50:05.643+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:50:05.645+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:50:05.645+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:50:05.653+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:50:05.651+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:50:05.654+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:50:05.681+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.049 seconds
[2025-03-31T21:50:36.340+0000] {processor.py:186} INFO - Started process (PID=2508) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:50:36.341+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:50:36.344+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:50:36.343+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:50:36.352+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:50:36.350+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:50:36.354+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:50:36.380+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.050 seconds
[2025-03-31T21:51:06.783+0000] {processor.py:186} INFO - Started process (PID=2572) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:51:06.785+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:51:06.788+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:51:06.788+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:51:06.797+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:51:06.795+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:51:06.800+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:51:06.832+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.061 seconds
[2025-03-31T21:53:28.770+0000] {processor.py:186} INFO - Started process (PID=2636) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:53:28.773+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:53:28.779+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:53:28.776+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:53:28.862+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:53:28.860+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:53:28.879+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:53:28.955+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.216 seconds
[2025-03-31T21:54:00.829+0000] {processor.py:186} INFO - Started process (PID=2699) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:54:00.832+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:54:00.835+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:54:00.835+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:54:00.896+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:54:00.890+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:54:00.910+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:54:00.992+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.199 seconds
[2025-03-31T21:55:10.124+0000] {processor.py:186} INFO - Started process (PID=2714) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:55:10.129+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:55:10.142+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:55:10.141+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:55:10.176+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:55:10.172+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:55:10.179+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:55:10.278+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.190 seconds
[2025-03-31T21:55:40.871+0000] {processor.py:186} INFO - Started process (PID=2776) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:55:40.873+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:55:40.884+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:55:40.883+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:55:40.906+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:55:40.898+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:55:40.907+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:55:41.011+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.158 seconds
[2025-03-31T21:56:11.123+0000] {processor.py:186} INFO - Started process (PID=2838) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:56:11.126+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:56:11.130+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:56:11.129+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:56:11.148+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:56:11.146+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:56:11.149+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:56:11.182+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.070 seconds
[2025-03-31T21:57:08.669+0000] {processor.py:186} INFO - Started process (PID=2857) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:57:08.671+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:57:08.674+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:57:08.673+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:57:08.727+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:57:08.724+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:57:08.730+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:57:08.799+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.145 seconds
[2025-03-31T21:57:41.003+0000] {processor.py:186} INFO - Started process (PID=2920) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:57:41.006+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:57:41.011+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:57:41.010+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:57:41.026+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:57:41.021+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.crawler import
                               ^
SyntaxError: invalid syntax
[2025-03-31T21:57:41.028+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:57:41.117+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.140 seconds
[2025-03-31T21:57:51.011+0000] {processor.py:186} INFO - Started process (PID=2963) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:57:51.012+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:57:51.016+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:57:51.015+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:57:51.032+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:57:51.030+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.download_data import
                                     ^
SyntaxError: invalid syntax
[2025-03-31T21:57:51.034+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:57:51.086+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.086 seconds
[2025-03-31T21:58:21.348+0000] {processor.py:186} INFO - Started process (PID=3009) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:58:21.353+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:58:21.371+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:58:21.370+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:58:21.447+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:58:21.445+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.download_data import
                                     ^
SyntaxError: invalid syntax
[2025-03-31T21:58:21.451+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:58:21.541+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.216 seconds
[2025-03-31T21:59:06.627+0000] {processor.py:186} INFO - Started process (PID=3054) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:59:06.633+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:59:06.637+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:59:06.636+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:59:06.693+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:59:06.679+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.download_data import
                                     ^
SyntaxError: invalid syntax
[2025-03-31T21:59:06.718+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:59:06.821+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.207 seconds
[2025-03-31T21:59:37.333+0000] {processor.py:186} INFO - Started process (PID=3115) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:59:37.335+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T21:59:37.338+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:59:37.337+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:59:37.347+0000] {logging_mixin.py:190} INFO - [2025-03-31T21:59:37.345+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.download_data import
                                     ^
SyntaxError: invalid syntax
[2025-03-31T21:59:37.348+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T21:59:37.391+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.068 seconds
[2025-03-31T22:00:07.655+0000] {processor.py:186} INFO - Started process (PID=3178) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T22:00:07.657+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T22:00:07.660+0000] {logging_mixin.py:190} INFO - [2025-03-31T22:00:07.659+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T22:00:07.688+0000] {logging_mixin.py:190} INFO - [2025-03-31T22:00:07.683+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.download_data import
                                     ^
SyntaxError: invalid syntax
[2025-03-31T22:00:07.690+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T22:00:07.737+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.096 seconds
[2025-03-31T22:00:37.800+0000] {processor.py:186} INFO - Started process (PID=3241) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T22:00:37.803+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T22:00:37.807+0000] {logging_mixin.py:190} INFO - [2025-03-31T22:00:37.806+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T22:00:37.817+0000] {logging_mixin.py:190} INFO - [2025-03-31T22:00:37.815+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.download_data import
                                     ^
SyntaxError: invalid syntax
[2025-03-31T22:00:37.819+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T22:00:37.871+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/premier_league_dag.py took 0.084 seconds
[2025-03-31T22:01:08.731+0000] {processor.py:186} INFO - Started process (PID=3304) to work on /opt/airflow/dags/premier_league_dag.py
[2025-03-31T22:01:08.746+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/premier_league_dag.py for tasks to queue
[2025-03-31T22:01:08.859+0000] {logging_mixin.py:190} INFO - [2025-03-31T22:01:08.834+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/premier_league_dag.py
[2025-03-31T22:01:09.198+0000] {logging_mixin.py:190} INFO - [2025-03-31T22:01:09.089+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/premier_league_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1133, in get_code
  File "<frozen importlib._bootstrap_external>", line 1063, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/premier_league_dag.py", line 1
    from scripts.download_data import
                                     ^
SyntaxError: invalid syntax
[2025-03-31T22:01:09.239+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/premier_league_dag.py
